src/HIE/Bios/Ghc/Check.hs

```diff
@@ -28,6 +27,8 @@ import qualified HIE.Bios.Types as T
 import qualified HIE.Bios.Ghc.Load as Load
 import HIE.Bios.Environment
 
+import Prettyprinter
+
```

Comment:
I think, so far, we used the old style import because @pepeiborra supports, unofficially, HLS for a GHC 8.8.4 stackage snapshot, which only has the old prettyprinter version. Updating this breaks HLS on a GHC 8.8.4. @pepeiborra Is that still relevant?